src/components/editor/sidebar/export-markdown-sidebar-entry.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useCallback } from 'react'
+import { useTranslation , Trans } from 'react-i18next'
+
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../redux'
+import { download } from '../../common/download/download'
+import { SidebarButton } from './sidebar-button'
+
+export const ExportMarkdownSidebarEntry: React.FC = () => {
+  useTranslation()
+
+  const markdownContent = useSelector((state: ApplicationState) => state.documentContent.content)
+  const onClick = useCallback(() => {
+    download(markdownContent, `title.md`, 'text/markdown')
```

Comment:
The note-title is hardcoded here.

Reply:
As discussed:
I won't change this, because this can be done as soon as the redux state provides the note title.